drivers/README.md

```diff
@@ -218,6 +218,8 @@ Some CCD drivers support streaming and recording of video streams. All the prope
 |                    |        | RECORD_DURATION_ON       | Start recording until the duration set in `RECORD_OPTIONS` has elapsed.               |
 |                    |        | RECORD_FRAME_ON          | Start recording until the number of frames set in `RECORD_OPTIONS` has been captured. |
 |                    |        | RECORD_OFF               | Stops recording.                                                                      |
+| Fast Exposure      | Switch | CCD_FAST_TOGGLE          | Fast Exposure is used to enable camera to immediately begin capturing the next frames.|
```

Comment:
That's not correct, the property name is CCD_FAST_TOGGLE, it has two element INDI_ENABLED and INDI_DISABLED. Check the code and revise the description.